Random Users/Views/UserCell.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  UserCell.swift
+//  Random Users
+//
+//  Created by Jeffrey Santana on 9/6/19.
+//  Copyright © 2019 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class UserCell: UITableViewCell {
+	
+	//MARK: - IBOutlets
+	
+	@IBOutlet weak var imgView: UIImageView!
+	@IBOutlet weak var nameLbl: UILabel!
+	
+	//MARK: - Properties
+	
+	var user: User? {
+		didSet {
+			configCell()
+		}
+	}
+	var imgData: Data?
+	
+	//MARK: - IBActions
+	
+	
+	//MARK: - Helpers
+	
+	private func configCell() {
+		guard let user = user else { return }
+		
+		imgView.loadImage(from: user.picture.thumbnail)
```

Comment:
if you comment this line (`imgView.loadImage(from: user.picture.thumbnail)`) the photos cease to load. This is using the `Data(contentsOf:)` extension you wrote and is not utilizing the `FetchPhotoOperation` at all. 

The fix is quite easy and involves your `imgData` property.

Reply:
forgot to remove that, but bug fixed by creating a computed property of the image data to run code when it is set

Reply:
perfect! That's exactly what I was hoping you'd do!